src/app/rpmostree-builtin-status.c

```diff
@@ -413,6 +413,34 @@ print_origin_repos (gboolean host_endian,
     }
 }
 
+static gboolean
+print_live_pkgdiff (const char *live_target, RpmOstreeDiffPrintFormat format, guint max_key_len, GCancellable *cancellable, GError **error)
+{
+  g_autoptr(OstreeSysroot) sysroot = ostree_sysroot_new_default ();
+  if (!ostree_sysroot_load (sysroot, cancellable, error))
+    return FALSE;
+  g_autoptr(OstreeRepo) repo = NULL;
+  if (!ostree_sysroot_get_repo (sysroot, &repo, cancellable, error))
+    return FALSE;
+  g_autoptr(GPtrArray) deployments = ostree_sysroot_get_deployments (sysroot);
+  OstreeDeployment *booted_deployment = ostree_sysroot_get_booted_deployment (sysroot);
+  g_assert (booted_deployment);
+
+  const char *from_rev = ostree_deployment_get_csum (booted_deployment);
+
+  g_autoptr(GPtrArray) removed = NULL;
+  g_autoptr(GPtrArray) added = NULL;
+  g_autoptr(GPtrArray) modified_old = NULL;
+  g_autoptr(GPtrArray) modified_new = NULL;
+  if (!rpm_ostree_db_diff (repo, from_rev, live_target,
+                          &removed, &added, &modified_old, &modified_new,
+                          cancellable, error))
+    return FALSE;
```

Comment:
Hmm, I think we should just gracefully allow errors here in case `live_target` doesn't exist anymore. We try to keep it around during rpm-ostree's cleanup, but it should be fine if it gets blown away somehow (e.g. if done via `ostree admin cleanup` instead). That way, we still get a status printout.